Gemfile

```diff
@@ -0,0 +1,54 @@
+source 'https://rubygems.org'
+git_source(:github) { |repo| "https://github.com/#{repo}.git" }
+
+ruby '2.6.5'
+
+# Bundle edge Rails instead: gem "rails", github: "rails/rails"
+gem "rails", "~> 6.0.3", ">= 6.0.3.2"
+# Use sqlite3 as the database for Active Record
+gem 'sqlite3', '~> 1.4'
+# Use Puma as the app server
+gem 'puma', '~> 4.1'
+# Use SCSS for stylesheets
```

Comment:
Remove all the comment lines
![image](https://user-images.githubusercontent.com/65896888/91243417-2d45e780-e774-11ea-9c1d-6ab06621b006.png)